ci/tests/viz/rpc-viz/src/entity/test.rs

```diff
@@ -9,13 +12,136 @@ fn test_to_from_proto() {
     let proto = entity.to_proto();
 
     match IdEntity::from_proto(&proto) {
-        Err(err) => {
-            panic!(
-                "Could not decode proto just created: {}. Proto: {:?}",
-                err,
-                proto
-            )
-        }
+        Err(err) => panic!(
```

Comment:
Instead of this do

```rust
let proto = entity.to_proto().expect(
    &format!("Could not decode proto just created: {}. Proto {:?}", err, proto)
 );
```

Reply:
This was one you wrote but I'll change :^)

Reply:
I'm dumb